src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -294,6 +294,35 @@
     @Parameter(defaultValue = "false", property = "formatter.useEclipseDefaults")
     private boolean useEclipseDefaults;
 
+    /**
+     * A java regular expression pattern that can be used to exclude some portions of the java code from being
+     * reformatted.
+     *
+     * This can be useful when using DSL that embeds some kind of semantic hierarchy, where users can use various
+     * indentation level to increase the readability of the code. Those semantics are ignored by the formatter, so this
+     * regex pattern can be used to match certain portions of the code so that they will not be reformatted.
+     *
+     * An example is the Apache Camel java DSL which can be used in the following way: <code><pre>
+     * 	from("seda:a").routeId("a")
+     * 			.log("routing at ${routeId}")
+     * 			.multicast()
+     * 				.to("seda:b")
+     * 				.to("seda:c")
+     * 			.end()
+     * 			.log("End of routing");
+     * </pre></code> In the above example, the exercept can be skipped by the formatter by defining the following
+     * property in the formatter xml configuration: <code>
+     * &lt;javaExclusionPattern>\b(from\([^;]*\.end[^;]*?\)\));&lt;/javaExclusionPattern>
+     * </code>
+     *
+     * Note that this property can also be set using the <code>net.revelc.code.formatter.java.exclusion_pattern</code>
+     * property in the xml configuration file for the java formatter.
```

Comment:
I'm very uncomfortable adding a `net.revelc.code.formatter.java.exclusion_pattern` config option to be parsed out of the Eclipse XML configuration file, as it creates a non-standard configuration file that conflates the Eclipse formatter's configuration with this plugin's configuration. It would also cause the XML to fail to be validated by any XSD that validates Eclipse config files.

Reply:
No problem, I've removed it.

Reply:
Thank you.